### PR TITLE
Add some more nightly tests

### PR DESCRIPTION
**Pull request by WouterJ**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,6 +6,7 @@ php:
   - 5.5
   - 5.6
   - hhvm
+  - nightly
```
Comment (dbu) [inline .travis.yml]: in other projects, we have `- 7` too.

**Comment by dbu** [comment]
the build fails. i guess this is about https://github.com/jackalope/jackalope-doctrine-dbal/pull/275 ?

**Comment by WouterJ** [comment]
@dbu Yes,

**Comment by dbu** [comment]
restarted the build

**Comment by lsmith77** [comment]
we need a new sonata version https://travis-ci.org/symfony-cmf/BlockBundle/jobs/61795138#L123

File: .travis.yml (hunk 1 above)
Comment (WouterJ) [inline .travis.yml]: nightly = 7

**Comment by WouterJ** [comment]
@lsmith77 it's more that 3.0 shouldn't be in the required-to-pass list of builds.

**Pull request merged**